bin/promotion.ml

```diff
@@ -30,6 +30,24 @@ let display_files files_to_promote =
     Console.printf "%s" (Diff_promotion.File.source file |> Path.Source.to_string))
 ;;
 
+let show_corrected_contents files_to_promote =
+  let open Fiber.O in
+  let files = Diff_promotion.load_db () |> Diff_promotion.filter_db files_to_promote in
+  Fiber.parallel_iter files ~f:(fun file ->
+    let correction_file = Diff_promotion.File.correction_file file in
+    let+ () = Fiber.return () in
+    if Path.exists correction_file
+    then (
+      let contents = Io.read_file correction_file in
+      Console.printf "%s" contents)
+    else
+      User_warning.emit
```

Comment:
Probably best to error here rather than warn. We only warn for `promote` because we didn't want to break backwards comparability.